tests/ssg_test_suite/rule.py

```diff
@@ -340,4 +341,25 @@ def perform_rule_check(options):
     checker.benchmark_cpes = options.benchmark_cpes
     checker.scenarios_regex = options.scenarios_regex
 
-    checker.test_target(options.target)
+    if options.target_source == 'profile':
+        if len(options.target) > 1:
+            logging.error("Specify only one target profile to "
+                          "scan in rule mode.")
+            return
+
+        profile = options.target[0]
+        # check if target is a complete profile ID, if not prepend profile prefix
+        if not profile.startswith(OSCAP_PROFILE):
+            profile = OSCAP_PROFILE+profile
+        logging.info("Performing rule test using profile: {0}".format(profile))
+
+        # Fetch target list from rules selected in profile
+        target_rules = xml_operations.get_all_rule_ids_in_profile(
+                options.datastream, options.benchmark_id,
+                profile, logging)
+        logging.debug("Profile {0} expanded to following list of "
+                      "rules: {1}".format(profile, target_rules))
```

Comment:
@yuumasato I have a question: What happens if some of the rules don't have any test scenarios?

Reply:
The rule won't be tested.

Reply:
I will look into writing a message when a rule is not tested.

Reply:
The `CombinedChecker` will output a list of rules not tested at the end of its run.